packages/next/build/webpack/loaders/error-loader.ts

```diff
@@ -1,12 +1,29 @@
+import chalk from 'chalk'
 import loaderUtils from 'loader-utils'
+import path from 'path'
 import { loader } from 'webpack'
 
 const ErrorLoader: loader.Loader = function() {
   const options = loaderUtils.getOptions(this) || {}
 
   const { reason = 'An unknown error has occurred' } = options
 
-  const err = new Error(reason)
+  // TODO: remove this ignore -- currently an ESLint bug
+  // eslint-disable-next-line @typescript-eslint/no-use-before-define
+  const resource = this._module?.issuer?.resource ?? null
+  // TODO: remove this ignore -- currently an ESLint bug
+  // eslint-disable-next-line @typescript-eslint/no-use-before-define
+  const context = this.rootContext ?? this._compiler?.context
+
+  const issuer = resource
+    ? context
+      ? path.relative(context, resource)
+      : resource
+    : null
+
+  const err = new Error(
+    reason + (issuer ? `\nLocation: ${chalk.cyan(issuer)}` : '')
```

Comment:
```suggestion
    reason + (issuer ? `\nLocation: ${chalk.cyan(issuer.replace(/\\/g, '/'))}` : '')
```
Do we wanna make this consistent for windows, if not we could update the tests to match both

Reply:
Updated the tests instead